testing/test_terminal.py

```diff
@@ -1199,15 +1219,6 @@ def test_summary_stats(exp_line, exp_color, stats_arg):
     assert color == exp_color
 
 
-def test_no_trailing_whitespace_after_inifile_word(testdir):
```

Comment:
This test is now unnecessary, `test_header` already covers this.

Reply:
Great.